src/kind.ts

```diff
@@ -46,7 +49,9 @@ export class KindConfig {
         if (this.waitDuration != "") {
             args.push("--wait", this.waitDuration);
         }
-
+        if (this.kubeConfigFile != "") {
+            args.push("--kubeconfig", this.kubeConfigFile);
```

Comment:
I assume the kubeconfig file is not intended to be pushed to the repository, right?
Would it be a secret? Where is it coming from?

Reply:
This only provides the location of where kubeconfig file is to `kind` so that it can update it. 


> I assume the kubeconfig file is not intended to be pushed to the repository


I'm not sure what you mean by `repository` here. Sorry if this is some basic knowledge, I'm new to using github actions.

Reply:
Take the Kind config file for example.
It is a configuration file that a user of this action would push to their GitHub repository, and the action would have access to (setting `with: config` when using the action to point to the path relative to the repo root).

If this is the kubeconfig path that Kind updates, then I assume it makes sense to not translate the path to be relative to the repository root.

Does that make sense?

Reply:
I got your point. 

> If this is the kubeconfig path that Kind updates, then I assume it makes sense to not translate the path to be relative to the repository root.
> 

Yes you are right. We do not need to translate the path. So, I think the current implementation is correct.